### DISCUSSION TITLE
通过vbs脚本实现开机自启并隐藏窗口的方法

### DISCUSSION THREAD

**Discussion by CloudSail1** [Simple-Tracker/qBittorrent-ClientBlocker#41] [General]
在程序目录创建.vbs文件，内容见下文，并创建快捷方式放入`C:\Users\YourUserName\AppData\Roaming\Microsoft\Windows\Start Menu\Programs\Startup`中即可。
```
Set ws = CreateObject("Wscript.shell")
ws.run "cmd /c qBittorrent-ClientBlocker.exe --startdelay 10",vbhide
```
删去`,vbhide`可以可以显示窗口。
startdelay参数会延迟10秒再启动，可以防止在代理软件启动前启动，导致fetch blocklist失败。

参考了csdn的文章：
https://blog.csdn.net/qq_35385687/article/details/129708891

#37 此方法亦可满足需求

**Comment by Simple-Tracker** [comment]
感谢提交!

已将 Issue 转换为 Discussion, 这是专供于用户讨论/分享使用的板块.
